@types/interfaces.d.ts

```diff
@@ -0,0 +1,30 @@
+import { ComboBoxElement } from '../src/vaadin-combo-box.js';
+
+export type ComboBoxItem = string | { [key: string]: unknown };
+
+export interface ComboBoxRendererContext {
```

Comment:
Not related to context menu. I wanted to keep the names consistent with Grid which uses "context" instead of "model" (I think context was also used in some other repo, maybe). I think it would be good to rename model to context also in the JSDocs, but didn't want to do that in scope of this PR.

Reply:
Actually I'm not sure anymore.. maybe I was looking at context menu. I'll recheck.

Reply:
In Grid, we use either `model` or `rowData`, which is not quite consistent...

Reply:
My bad. I'll fix it and use "model".

Reply:
Updated.